### DISCUSSION TITLE
APA Format Template to Word Doc and/or PDF

### DISCUSSION THREAD

**Discussion by nb51** [quarto-dev/quarto-cli#4101] [Q&A]
I am new to R and Quarto, so I apologize if this is a simple question. I successfully created my first Quarto document in PDF format but wanted to know if there is any way to have it create a Word and/or PDF document in the newest APA (American Psychological Association) formatting (e.g., font, spacing, headings, graphs, tables, etc.)? This would help out with some homework and publication writing for my graduate program in psychology.

**Comment by cscheid** [comment]
I don't think we have an APA format yet (https://github.com/quarto-journals/). There seems to be RMarkdown support for APA, according to https://osf.io/mhc9p/download through the papaja package, which makes me hopeful that a quarto template wouldn't be too hard. If you're willing to try out, I'd be happy to assist you.

**Reply by dragonstyle** [reply]
This one is in progress - might also be able to chip in there:

https://github.com/alsonicr/quarto-apa7

**Reply by nb51** [reply]
> I don't think we have an APA format yet (https://github.com/quarto-journals/). There seems to be RMarkdown support for APA, according to https://osf.io/mhc9p/download through the papaja package, which makes me hopeful that a quarto template wouldn't be too hard. If you're willing to try out, I'd be happy to assist you.

That would be great. I am still pretty basic with my understanding of R and Quarto (used SPSS for some undergrad work before) but I think this formatting stuff would help tremendously with my data analysis/research efficiency and communication. Please let me know what I can do

**Reply by cderv** [reply]
We have a feature request for it I believe https://github.com/quarto-journals/article-format-template/issues/11 and `papaja` are working on https://github.com/crsh/qapaja.

**Comment by alsonicr** [comment]
I did produce one here [https://github.com/alsonicr/quarto-apa7](https://github.com/alsonicr/quarto-apa7)

**Comment by wjschne** [comment]
Here is a quarto extension I made:
https://github.com/wjschne/apaquarto

It creates APA7 documents in .docx, .html, and .pdf formats.
